packages/anvil/Dockerfile

```
@@ -0,0 +1,20 @@
+# syntax=docker.io/docker/dockerfile:1.4
+FROM debian:bullseye-20230320-slim
```

```suggestion
FROM debian:bullseye-20230502-slim
```

see: 

- https://github.com/docker-library/official-images/pull/14557
- https://hub.docker.com/_/debian/tags?page=1&name=bullseye-20230502-slim

---

Changed debian version and squashed.